README.md

```diff
@@ -25,22 +25,26 @@ _Eventually a bootstrap process will be handled by an [ado terraform provider](h
   - https://dev.azure.com/<your_organization>/osdu-r2/_git/osdu-r2
 
 - Create Empty Repositories (No Readme)
-  - osdu-infrastructure
```

Comment:
So the process I have is also syncing the infrastructure changes not just the service code repositories.  I know this isn't mandatory to get things to run but I think it may be important and here is why.  A change is going in right now to expand the capability of entitlements.  This required a change also to the infrastructure to support it.  

Additionally another example would since delivery is not yet completed on Azure infrastructure does not currently deploy a delivery web app.

Agree/Disagree?

Reply:
Agree